src/modules/TeamsList/components/Teams/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from 'react';
+import { PageTitle } from 'typography';
+
+import { TeamsHeader } from './components/TeamsHeader/TeamsHeader';
```

Comment:
change filename to index.tsx and import as import { TeamsHeader } from './components/TeamsHeader';

Reply:
done